bpf/tests/xdp_nodeport_lb4_nat_lb.c

```diff
@@ -62,8 +62,10 @@ long mock_fib_lookup(__maybe_unused void *ctx, struct bpf_fib_lookup *params,
 	__u32 key = 0;
 	struct mock_settings *settings = map_lookup_elem(&settings_map, &key);
 
-	if (settings && settings->fail_fib)
+	if (settings && settings->fail_fib) {
+		params->ifindex = 1;
 		return BPF_FIB_LKUP_RET_NO_NEIGH;
```

Comment:
Let's introduce this here as well, and refer to it further below:

```suggestion
		params->ifindex = DEFAULT_IFACE;
		return BPF_FIB_LKUP_RET_NO_NEIGH;
```

Reply:
Done.

Originally, `NATIVE_DEV_MAC_BY_IFINDEX` had a default mac. This is to lookup the smac of the interface.
```
#define NATIVE_DEV_MAC_BY_IFINDEX(_) { .addr = { 0xce, 0x72, 0xa7, 0x03, 0x88, 0x56 } }
```

Without it, the `xdp_nodeport_nat_fwd_reply_no_fib` test will fail. the error is `src MAC is not the LB MAC`.